frontend/package.json

```diff
@@ -51,8 +49,6 @@
     "eslint-formatter-summary": "^1.1.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-vue": "^8.1.1",
-    "husky": "^7.0.4",
-    "lint-staged": "^12.1.2",
     "prettier": "^2.4.1",
```

Comment:
Need to uninstall husky in your local env, and re-install `pre-commit` by

```
git config --unset-all core.hooksPath
pre-commit install
pre-commit install --hook-type commit-msg
```

otherwise `pre-commit` won't work cased by husky hook

Reply:
And don't forget to remove `./frontend/.husky` folder in local